canonicalwebteam/search/views.py

```diff
@@ -46,18 +44,16 @@ def build_search_view(
         )
     """
 
+    app = flask.Flask(__name__)
```

Comment:
This doesn't work also because you are creating and overriding an exisiting Flask app. So you need to
```
from flask import current_app
.....

limiter = Limiter(
        get_remote_address, app=current_app, default_limits=[request_limit]
    )

```

Reply:
@petesfrench this one in particular